gorp.go

```diff
@@ -257,15 +257,15 @@ func columnToFieldIndex(m *DbMap, t reflect.Type, name string, cols []string) ([
 			cArguments := strings.Split(field.Tag.Get("db"), ",")
 			fieldName = cArguments[0]
 
+			if fieldName == "" || fieldName == "-" {
```

Comment:
Just wanted to clarify quickly: is this change already covered by the change in the unit tests? If not, can we add some tests to cover them?

Reply:
Yes it is covered but I’ll add more explicit tests to be really sure